Makefile

```
@@ -18,11 +18,11 @@ PREV_RELEASE=release-1.26
 # any images we need based on commit are matched/substituted correctly.
 CEPH_CSI_COMMIT=47b59ee5a430f66a88913bea1a6ac1961c8ff552 # v3.7.2
 COREDNS_COMMIT=31e9b6e2229300280f9788b1eaf1eb18c1b2d5c6 #v1.9.4
-OPENSTACK_PROVIDER_COMMIT=58b9f9fe82a4e3a697991d68d621b78da75771b0 # v1.25.3
-K8S_KEYSTONE_AUTH_IMAGE_VER=v1.25.3  # override keystone auth image
+OPENSTACK_PROVIDER_COMMIT=afc4309cbc84c70d475d9f16bc24cd0d5e9ea728 # v1.26.2
+K8S_KEYSTONE_AUTH_IMAGE_VER=v1.26.2  # override keystone auth image
```

this is so weird... i'm seeing a different commit sha

https://github.com/kubernetes/cloud-provider-openstack/commits/v1.26.2
```suggestion
OPENSTACK_PROVIDER_COMMIT=fab51c7b80b6b7067dad73b156da93b932a70353 # v1.26.2
K8S_KEYSTONE_AUTH_IMAGE_VER=v1.26.2  # override keystone auth image
```

---

perhaps it's because i'm looking at the tag `1.26.2` and not the [`release-1.26` branch](https://github.com/kubernetes/cloud-provider-openstack/commits/release-1.26)